lib/ansible/modules/cloud/amazon/sns_topic.py

```diff
@@ -23,6 +23,7 @@
   - "Joel Thompson (@joelthompson)"
   - "Fernando Jose Pando (@nand0p)"
   - "Will Thames (@willthames)"
+  - "Brian Huang (@brianhhq)"
```

Comment:
```suggestion
```
 Thanks very much for this new feature. In the Ansible project, module authors are our community maintainers who are granted the ability to issue [shipits](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md#community) for changes, basically giving these individuals merge powers. As this is not a new module or a substantial rewrite, this line isn't needed here. We'd like to have you get more involved with the project before granting approval capabilities on modules.
Are there certain modules you are particularly interested in?
Reviewing PRs really helps us see [AWS priorities](https://github.com/ansible/community/wiki/AWS%3A-Pull-Request-Priorities) for a list of GitHub searches.